### PR TITLE
[RFR]Fix mutable args

### PR DESCRIPTION
**Pull request by psav**
Though some of these are not necessary, it would perhaps be better to
set a standard of no mutable args. These changes shouldn't break
anything and are really to keep QC happier.

### CODE REVIEW

File: cfme/control/explorer/actions.py [hunk 1]
```diff
@@ -119,8 +119,9 @@ class Action(Updateable, Navigatable, Pretty):
         description: Action name.
         action_type: Type of the action, value from the dropdown select.
     """
-    def __init__(self, description, action_type, action_values={}, appliance=None):
+    def __init__(self, description, action_type, action_values=None, appliance=None):
         Navigatable.__init__(self, appliance=appliance)
+        action_values = {} if not action_values else action_values
```
Comment (mfalesni) [inline cfme/control/explorer/actions.py]: `action_values = action_values or {}`

File: utils/bz.py [hunk 2]
```diff
@@ -135,8 +135,10 @@ def get_bug_variants(self, id):
                 expanded.add(b)
         return found
 
-    def resolve_blocker(self, blocker, version=None, ignore_bugs=set([]), force_block_streams=[]):
+    def resolve_blocker(self, blocker, version=None, ignore_bugs=None, force_block_streams=None):
         # ignore_bugs is mutable but is not mutated here! Same force_block_streams
+        force_block_streams = [] if not force_block_streams else force_block_streams
+        ignore_bugs = set([]) if not ignore_bugs else ignore_bugs
```
Comment (mfalesni) [inline utils/bz.py]: `ignore_bugs = ignore_bugs or set()`

**Comment by psav** [comment]
It is, I went into silly mode ;)

**Comment by psav** [comment]
Fixed

**Pull request merged**